assignments/array-methods.js

```diff
@@ -56,28 +56,91 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach((arr) => {
```

Comment:
`arr` is a confusing name for a placeholder, since `runners` is not an array of arrays (it is an array of objects)